### ISSUE TITLE
[bug #10329] crtm168.o is not being built

### ISSUE THREAD

**Issue by avrs-admin** [avrdudes/avr-libc#69]
Thu 09 Sep 2004 12:41:53 AM CEST

Built gcc 3.4.2 with patch to add new devices. In building avr-libc 1.0.4, crtm168.o is not built.

I found a typo in configure.in that would cause crtm168.o to not get added to the AVR_CRT_MEGA list. Patch is attached to correct this. But this does not completely solve the problem.

I've set this priority to 9 because it's blocking a user from using this device.

Ted, could you take a look at this? You're more familiar with the configury than I am. Thanks.

Eric

[file #1716: avr-libc-mega168-crt.patch](https://savannah.nongnu.org/bugs/download.php?file_id=1716)

This issue was migrated from https://savannah.nongnu.org/bugs/?10329

**Comment by avrs-admin** [comment]
Thu 09 Sep 2004 12:58:00 AM CEST

The typo was fixed a long time ago:

[http://savannah.nongnu.org/cgi-bin/viewcvs/avr-libc/avr-libc/configure.in.diff?r1=1.111&r2=1.112](https://savannah.nongnu.orghttp://savannah.nongnu.org/cgi-bin/viewcvs/avr-libc/avr-libc/configure.in.diff?r1=1.111&r2=1.112)

As for the crt file not being built, what does configure say when it check for atmega168 support in gcc? If that check fails, then the crt will not be built.

**Comment by avrs-admin** [comment]
Eric Weddington <arcanum>
Fri 10 Sep 2004 11:43:51 PM CEST

Other than the problem with the AVR_CRT_MEGA list in configure.in, which I was unaware that it was fixed earlier in CVS but it is not in any released version, this was a user error (me).

Conflicting experiments I was doing caused the avr-gcc that I built to find the incorrect avr-as (from an earlier version) and would not subsequently build crtm168.o. This is now fixed on my end.

Thanks for your help Ted!

Eric

**Comment by avrs-admin** [comment]
Eric Weddington <arcanum>
Fri 10 Sep 2004 11:53:47 PM CEST

Other than the problem with the AVR_CRT_MEGA list in configure.in, which I was unaware that it was fixed earlier in CVS but it is not in any released version, this was a user error (me).

Conflicting experiments I was doing caused the avr-gcc that I built to find the incorrect avr-as (from an earlier version) and would not subsequently build crtm168.o. This is now fixed on my end.

Thanks for your help Ted!

Eric
